modules/gcp_bigquery_big_lake_table/inputs.tf

```diff
@@ -21,6 +22,7 @@ variable "dataset_kms_key_name" {
 variable "deletion_protection" {
   type        = bool
   description = "Whether or not to prevent Terraform from destroying the instance."
+  default     = false
```

Comment:
default should be true.

Reply:
Not necessary for BigLake tables. These tables do not contain any data and re-creating them is fine. But since I'm already making changes here, I will set this to true on the native BQ table.